lib/vmdb/loggers/audit_logger.rb

```diff
@@ -1,13 +1,15 @@
-module Vmdb::Loggers
-  class AuditLogger < VMDBLogger
-    def success(msg)
-      info("Success") { msg }
-      $log.info("<AuditSuccess> #{msg}") if $log
-    end
+module Vmdb
+  module Loggers
+    class AuditLogger < VMDBLogger
+      def success(msg)
+        info("Success") { msg }
+        $log.info("<AuditSuccess> #{msg}") if $log
+      end
 
-    def failure(msg)
-      warn("Failure") { msg }
-      $log.warn("<AuditFailure> #{msg}") if $log
+      def failure(msg)
+        warn("Failure") { msg }
+        $log.warn("<AuditFailure> #{msg}") if $log
+      end
```

Comment:
@kbrock I understand the other changes but why do we need to expand the collapsed namespace syntax?  I don't see a referenced error this collapsed syntax was causing.  I personally dislike the collapsed syntax but I can't figure out why this was a problem.  Missing constant?  What am I missing?